solidity/test/LPRewardsTest.js

```diff
@@ -0,0 +1,119 @@
+const {contract, web3} = require("@openzeppelin/test-environment")
+const {createSnapshot, restoreSnapshot} = require("./helpers/snapshot")
+const {accounts} = require("@openzeppelin/test-environment")
+
+const LPRewards = contract.fromArtifact("LPRewards")
+const KeepToken = contract.fromArtifact("KeepToken")
+const WrappedToken = contract.fromArtifact("TestToken")
+
+const BN = web3.utils.BN
+const chai = require("chai")
+chai.use(require("bn-chai")(BN))
+const expect = chai.expect
+
+describe("LPRewards", () => {
```

Comment:
What flows do we want to cover with tests?

Reply:
What I wanted to add are some happy paths that can help us better understand the flows in these contracts:
- KEEP rewards allocations
- Wrapped Token (lp tokens) allocations
- User's KEEP rewards earnings after staking
- User's KEEP rewards and Wrapped tokens withdrawals